### PR TITLE
added route endpoints

### PR DESCRIPTION
**Pull request by GigiDev7**

### CODE REVIEW

**Comment by GigiDev7** [comment]
Hello. I dont know exactly what kind of data i will have or what kind of collections i will have in Mongo so maybe i will have to add some more endpoints after that or remove existing.

**Comment by GigiDev7** [comment]
I saw now that i dont use authentication router anywhere and i have get  method on login , also i dont neeed logout route, i will change that

**Comment by GigiDev7** [comment]
I have bug i should have post request on login...

File: controllers/account.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const getAllAccounts = (req, res) => {
```
Comment (Lokki17) [inline controllers/account.js:1]: It's better to split logic by layers (controllers, services, data)
Controllers are used to handle request, services to store busines logic and data-layer to store data. In your case, I think, service and data will be combined.
In general cases services shouldnt know about requests.
Just move busines logic to another file.

File: controllers/expanses.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const getExpanses = (req, res) => {
```
Comment (Lokki17) [inline controllers/expanses.js:1]: the same remark as for account

File: controllers/incomes.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const getIncomes = (req, res) => {
```
Comment (Lokki17) [inline controllers/incomes.js:1]: the same remark as for account

**Review by Lokki17 — CHANGES_REQUESTED** [review]
Please fix remarks

File: controllers/account.js (hunk 1 above)
Comment (GigiDev7) [reply controllers/account.js:1]: Hello, for data I will have models folder like MVC architecture(but i won't have views in this project beecause we use client side rendering).

**Comment by GigiDev7** [comment]
As i understood i should have services folder where i put logics, like database queries, algorithms and stuff, then i import them in controllers and just call it in controller. i dont have any logic right now because i dont have any database or data to return but i will add services folder after i have it, i never used it but i think i guessed what i should do

**Comment by GigiDev7** [comment]
I made services folder and made example function, is it the way it should be? or i did not get it right?

**Comment by GigiDev7** [comment]
I'll add models folder for mongoose schemas after we start working with DB

File: controllers/account.js (hunk 1 above)
Comment (Lokki17) [reply controllers/account.js:1]: Ok

**Review by Lokki17 — APPROVED** [review]

**Comment by Lokki17** [comment]
> I made services folder and made example function, is it the way it should be? or i did not get it right?

Yes. It's correct

**Comment by GigiDev7** [comment]
Thank you!

**Pull request merged**